### DISCUSSION TITLE
How can I apply admonition boxes from third party to Jekyll posts?

### DISCUSSION THREAD

**Discussion by logankim0913** [cotes2020/jekyll-theme-chirpy#1195] [Q&A]
I am using Chirpy-themed Jekyll in VSCode with the "Markdown Extended" extension. It has a feature for admonition boxes with designs I would like to put on my posts. However, while I can see the admonition boxes in the markdown preview, Jekyll doesn't seem to render the admonition box when the post is put on the server. I checked the internet for quite a bit now, but I couldn't find a source that explicitly says that Jekyll does not support this particular extension. I have created my own admonition boxes by adding variables to `variables-hook.scss`, but they do not look appealing. Is there a way to apply the admonition boxes (or any features) from other parties to Chirpy? 

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/51704336/48d26e6b-b14a-4af2-96bc-4e9199e7c97e)
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/51704336/cedb7beb-c9c6-45d2-8325-a08b16645169)

**Comment by NichtsHsu** [comment]
Maybe https://chirpy.cotes.page/posts/text-and-typography/#prompts ?

**Reply by logankim0913** [reply]
Well, it does the job but the functionality is very limited. I was wondering how I can use the markdown functionalities from other parties with Chirpy.
